packages/patterns-react/src/patterns/ContentArrayWithPictograms/ContentArrayWithPictograms.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
+import { DDS_CONTENTARRAYWITHPICTOGRAMS } from '../../internal/FeatureFlags';
+import ContentArrayWithPictogramsItem from './ContentArrayWithPictogramsItem';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Content Array with Pictograms
+ *
+ * @param {object} props props object
+ * @param {string} props.title List section title
+ * @param {Array} props.contentGroup variation of the List section standard, standard with jump link and standard with card link
+ * @returns {*}  Contennt array with pictograms JSX Component
+ */
+const ContentArrayWithPictograms = ({ title, contentGroup }) =>
```

Comment:
Pretty lengthy name....maybe go with `PictogramArray`? Anyone else have thoughts?
cc: @RaphaelAmadeu @emerson-pereira @wonilsuhibm @ltitle @kennylam @annawen1 @andysherman2121 @larahanlon2  @photodow

Reply:
I guess `PictogramArray` will indeed do better.

Reply:
:+1: Yeah, I believe `PictogramArray` is where we last landed. Still not 100% on board, but at least it's the latest we have for now.